feature_layers/change-feature-layer-renderer/src/main/java/com/esri/samples/change_feature_layer_renderer/ChangeFeatureLayerRendererSample.java

```diff
@@ -61,13 +61,13 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create a blue (0xFF0000FF) line symbol renderer
-      SimpleLineSymbol lineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFF0000FF, 2);
-      SimpleRenderer blueRenderer = new SimpleRenderer(lineSymbol);
+      // create a marker symbol renderer with a green circle
+      SimpleMarkerSymbol markerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, 0xFF317D11, 5 );
+      SimpleRenderer blueRenderer = new SimpleRenderer(markerSymbol);
 
       // create renderer toggle switch
       ToggleButton rendererSwitch = new ToggleButton();
-      rendererSwitch.setText("blue renderer");
+      rendererSwitch.setText("green renderer");
```

Comment:
we should also set the text to display "original renderer" when the green renderer is displaying (so the user knows when clicking the button it will go back to original renderer)

Reply:
Changing the toggle button's text gives a bit of a strange UI, I find, so I swapped it with a tickbox that says 'apply green renderer'. Also updated the screenshot/gif to match. What do you think?